src/events/evaluationsEvents.js

```diff
@@ -0,0 +1,13 @@
+ //get id from each button on evaluations page
+ /*function evaluationsEvents(options = {}) {
```

Comment:
If no longer required, let's go ahead and remove altogether.

Reply:
it is needed but it doesn't work properly and I preferred to change the structure using modules before solving the problem